spec/support/fixtures/dummy_app/bin/yarn

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env ruby
+
+require "pathname"
```

Comment:
Differently from the fake `bundler` binary, this binary is located in the dummy app's fixtures. The reason is that we call `yarn` from the root of the Rails application, like this:  `bin/yarn` -- so `yarn` does not need to be in the `PATH`.